reference/docs-conceptual/whats-new/differences-from-windows-powershell.md

```diff
@@ -382,6 +399,124 @@ PS> SimpleTest @hash "MyPath"
 Name: Hello; Path: MyPath; Args: -Blah: World
 ```
 
+## Language changes
+
+### Null-coalescing operator `??`
+
+The null-coalescing operator `??` returns the value of its left-hand operand if
+it isn't null. Otherwise, it evaluates the right-hand operand and returns its
+result. The `??` operator doesn't evaluate its right-hand operand if the
+left-hand operand evaluates to non-null.
+
+```powershell
+$x = $null
+$x ?? 100
+```
+
+```Output
+100
+```
+
+In the following example, the right-hand operand won't be evaluated.
+
+```powershell
+[string] $todaysDate = '1/10/2020'
+$todaysDate ?? (Get-Date).ToShortDateString()
+```
+
+```Output
+1/10/2020
+```
+
+### Null-coalescing assignment operator `??=`
+
+The null-coalescing assignment operator `??=` assigns the value of its
+right-hand operand to its left-hand operand only if the left-hand operand
+evaluates to null. The `??=` operator doesn't evaluate its right-hand operand
+if the left-hand operand evaluates to non-null.
+
+```powershell
+$x = $null
+$x ??= 100
+$x
+```
+
+```Output
+100
+```
+
+In the following example, the right-hand operand won't be evaluated.
+
+```powershell
+[string] $todaysDate = '1/10/2020'
+$todaysDate ??= (Get-Date).ToShortDateString()
+```
+
+```Output
+1/10/2020
+```
+
+### Null-conditional operators `?.` and `?[]`
+
+> [!NOTE]
+> This feature was moved from experimental to mainstream in PowerShell 7.1.
```

Comment:
We have a note about version here but not for `??` or `??=` - should those be updated or should this note be removed, or is this intentional?

Reply:
Will address